docker-hub/download-rate-limit.md

```diff
@@ -26,8 +26,46 @@ manifest requests.
 - Limits are applied based on the user doing the pull, and 
 not based on the image being pulled or its owner.
 
-Docker will gradually introduce these rate limits, with full
-effects starting from November 1st, 2020.
+Docker will gradually introduce these rate limits starting November 2nd, 2020.
+
+## How do I know my pull requests are being limited
+
+When you issue a pull request and you are over the limit for your account type, Hub will return a `429` response code with the following body when the manifest is requested:
+> You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limits
+
+You will see this error message in the Docker CLI or Docker Engine logs.
+
+## How can I check my current rate
+
+Valid, non-rate-limited manifest API reqests to Hub will include the following rate limit headers in the response:
```

Comment:
> non-rate-limited manifest API requests to Hub
In testing we only return these headers for GET requests. Maybe we should return them for HEAD requests as well so that users can get information without increasing their rate limiting counter?

Reply:
For sure, let's file that as an improvement post-launch.

Reply:
There's a typo here, "reqests"

Reply:
```suggestion
Valid, non-rate-limited manifest API requests to Hub will include the following rate limit headers in the response:
```